server/routes/utilMongoose.js

```diff
@@ -18,6 +18,7 @@ const conversationSchema = require('../models/conversationSchema');
 const collegeRepSchema = require('../models/collegeRepSchema');
 require('mongoose').Promise = global.Promise
 var COUNTRIES = require("../countries").COUNTRIES
+const { v4: uuidv4 } = require('uuid');
```

Comment:
Changes to this file ensure that the /seed API does not fail during tests due to deplicateKeyErrors